doc/devdocs/tools/bug-report-tool.md

```diff
@@ -0,0 +1,31 @@
+# [Bug report tool](/tools/BugReportTool/)
+
+This tool is used to collect logs and system information for bug reports. The bug report is then saved as a zip file on the desktop.
+
+## Launching
+
+It can launch from the PowerToys tray icon by clicking "Report Bug" or by running the executable directly.
```

Comment:
```suggestion
It can launch from the PowerToys tray icon by clicking "Report Bug", by clicking the bug report icon in the PowerToys flyout  or by running the executable directly.
```